packages/core/src/Tab/index.tsx

```diff
@@ -154,12 +154,27 @@ export function TabBase<T>({
     onSelect(id)
   }, [selected, disabled, id, onSelect])
 
+  const handleKeyDown = useCallback<React.KeyboardEventHandler<BaseElement>>(
+    event => {
+      if (selected || disabled) {
+        return
+      }
+
+      if (event.key === 'Enter') {
+        onSelect(id)
+      }
+    },
+    [selected, disabled, id, onSelect]
+  )
+
   return (
     <TabBaseContainer
       disabled={disabled}
       onClick={onClick}
       selected={selected}
       markerOffset={markerOffset}
+      tabIndex={0}
+      onKeyDown={handleKeyDown}
       {...props}
```

Comment:
Move this above `onKeyDown`

Reply:
Done.

Reply:
Sorry, as you using destructuring now online `147` please make `{...props}` last as you had before 😃

Reply:
Moved down 😄.

Reply:
👍